core/src/main/scala/caliban/CalibanError.scala

```diff
@@ -55,25 +62,27 @@ private object ErrorCirce {
     Json.obj("line" -> li.line.asJson, "column" -> li.column.asJson)
 
   val errorValueEncoder: Encoder[CalibanError] = Encoder.instance[CalibanError] {
-    case CalibanError.ParsingError(msg, locationInfo, _) =>
+    case CalibanError.ParsingError(msg, locationInfo, _, extensions) =>
       Json
         .obj(
           "message" -> s"Parsing Error: $msg".asJson,
           "locations" -> Some(locationInfo).collect {
             case Some(li) => Json.arr(locationToJson(li))
-          }.asJson
+          }.asJson,
+          "extensions" -> extensions.asInstanceOf[Option[ResponseValue]].asJson.dropNullValues
```

Comment:
`(extensions: Option[ResponseValue])` would be better than `asInstanceOf` (here and below)

Reply:
Makes a lot of sense but I can't seem to get the encoder for an `Option[ResponseValue]` in implicit scope such that `.asJson` works. Thought that the implicit `circeEncoder` of `ResponseValue` would be able to pick this one up? I got the `.asInstanceOf[ResponseValue]` trick from `GraphQLResponse` actually.

Reply:
`(extensions: Option[ResponseValue]).asJson.dropNullValues` doesn't work? We were about to remove the `asInstanceOf` in this PR by using that same trick: https://github.com/ghostdogpr/caliban/pull/234/files#diff-16ab822f0162e843af921d1e9d0c34dfR23

Reply:
Aha, I was trying to put it in the match clause. Apparently didn't fully understand what you meant. Changed it now and it works!